modules/appeals_api/spec/requests/api_docs_request_spec.rb

```diff
@@ -4,20 +4,17 @@
 
 RSpec.describe 'Appeals Documentation Endpoints', type: :request do
   # @param [Hash] opts
-  # @option opts [String] :api_version The version of the OAS documentation to return. Required.
-  # @option opts [Hash] :doc_path The suffix path to retrieve the OAS json from. Required.
-  # @option opts [String] :server_path The OAS server suffix path. Required.
```

Comment:
Using the newer dashed URLs here allowed a lot of simplification here. The old routes are no longer used anywhere. I was thinking I'd remove them entirely soon - The only reason to still maintain any route aliases at the moment is for the renaming of contestable issues, otherwise I'd remove them now.